language-design.md

```diff
@@ -48,8 +58,11 @@ How do the themes of _Growing a Language_ relate to the "sound lab" we did this
 In what way is an API a language? 
 
 **Response**
+APIs can be categorized as internal domain-specific languages, since they are written inside an existing host language.
```

Comment:
I'm not convinced by this. Consider a web API. While I might implement the API endpoints in a language (let's say C#), users who write "code" in the API don't necessarily use C#; if anything they use the language of HTTP requests